plugins/Kaleidoscope-LEDControl/README.md

```diff
@@ -1,6 +1,123 @@
 # Kaleidoscope-LEDControl
 
 This is a plugin for [Kaleidoscope][fw], for controlling the LEDs, and LED
-effects.
+effects. It is also a building block for plugins that control LEDs.
 
  [fw]: https://github.com/keyboardio/Kaleidoscope
+
+## Using the extension
+
+## Plugin methods
+
+### `.next_mode(void)`
+
+> Activates the next LED mode. Cycles to the first LED mode if the current LED
+> mode is the last one.
+
+### `.prev_mode(void)`
+
+> Activates the previous LED mode. Cycles to the last LED mode if the current
+> LED mode is the first one.
+
+### `.get_mode()`
+
+> Returns the current LED mode.
+
+### `.get_mode<typename>()`
```

Comment:
I don't know how to properly explain this. Having looked at [the commit](https://github.com/keyboardio/Kaleidoscope/commit/f54e11a9b2604ec0fe2931549773bbec42815397) you mentioned on discord I understand what it does, but I don't know the words to explain it.